#Building a 16S rDNA Phylogenetic Tree

What _is_ this thing? At this point, you have an organism in pure culture, but you don't know what it is. If you found an creature crawling on the ground and
wanted to identify (or classify) it, you might look at it's morphology and ask what it most _looks_ like. If it has six legs, you might guess it's some kind of
insect. If it has hard outer wings folded over its back, you might guess that it's some kind of beetle. If it also had antler-style horns on its head, you might
guess that it's some kind of stag beetle. If you don't have enough information available to guess what kind of stag beetle you have, then you have reached the
limit of *taxonomic resolution* for your creature. With an unknown microbial species, the best way to identify it is to sequence one of its genes (most people
use the 16S rRNA gene) and ask what _it_ most looks like. With animal classification, the key features to examine are things like legs and wings and horns; with microbial
classification, the key features to examine are the nucleotides in different positions in a DNA sequence. Fortunately, we have computer programs to help us make
sense of the DNA sequence information. Our preferred approach to identifying a microbial species is to place the unknown sequence in the context of a
phylogenetic tree of known sequences. Building a phylogenetic tree from a 16S rRNA sequence is fairly straightforward, but the interpretation of the tree can be
a bit of an art. Here, we attempt to guide you through both. However, some complicated cases will require consultation with an expert in the field of
phylogenetics or systematics.

The outline of the workflow is to use the Ribosomal Database Project (RDP) to generate an alignment of the sequence with close relatives and an outgroup,
following by cleanup of the RDP headers, tree-building with FastTree and viewing/interpretation of the tree using Dendroscope.

##Obtain an RDP alignment

The goal of this section is to obtain a 16S alignment from RDP that can be used to build a tree.

1. Go to rdp.cme.msu.edu 
2. Create an account 
3. Click on my RDP/login 
4. Upload the fasta file containing your 16s sequence 
5. Assign it a group name (this is what the program will label your sequence/organism). Choose this carefully since that will be the name on the final tree. 
6. Click the "+" next to the sequence, to add it to your cart
7. Click on CLASSIFIER at the top of the page
8. Click on "Do Classification With Selected Sequences" button. This will show you a hierarchical view of the classification of your sequence (from Phylum to Genus.) You will use this information to navigate to other sequences that you want to include in your alignment that you will use to build your phylogenetic tree. For example, Figure X shows the Hierarchy for _Bacillus subtilus_. 
8. Click on BROWSERS. We recommend openining BROWSERS in a new tab so that you can keep the hierarchy information handy.
9. Click on the + sign next to "Archaea outgroup." This will add an Archaeal sequence to your cart, which will be used to root your phylogenetic tree.
10. If using the example sequence provided, click on Firmicutes, then Bacilli, then Bacillales, then Bacilaceae. This will take you to the Genus Bacillus, which currently has over 26,000 species in it. One way to reduce this number is to exclude the uncultured taxa in the database. To do this, scroll down to the Data Set Options and click on the "Isolates" button. Click "Refresh" and you will see that there are still more than 22,000 sequences in the Genus. To reduce this number further, click on the "Type" Strain button. This will result in a reasonable number of 181 taxa.
11. Click on the + sign next to **genus** Bacillus to add all of those sequences to your cart.
12. Click on TREE BUILDER
13. Specify your outgroup. At the time of publication, the Archaeal outgroup is Methanocaldococcus.
14. Click "Create Tree"


 
Note: RDP will not generate trees with more than 200 sequences. However, if following our recommendation of using Fasttree then this limit does not apply. You can check how many sequences are in your cart by looking to the right of "Hierarchy Browser".

In most cases the tree can be built with sequences from other isolates within the same genus. For a genus with relatively few sequences in RDP, this is fairly straightforward; select the entire genus and move on to choosing an outgroup. For genera with very large numbers of sequences, trimming is required. Since the goal is to discover the placement of a candidate sequence within the genus, the best pool of sequences would contain multiple sequences from each species in that genus. This requires manual selection of sequences in the Browser. We recommend using sequences with informative names.

In some rare cases the tree generated by RDP will be sufficient, other times it may fail to load, crash, or be impossible to resolve detail. We recommend instead, downloading the alignment and building the tree in FastTree.

Building the Tree with FastTree
Download the alignment you just created in RDP
 -Click on download, use the default settings (fasta, aligned, uncorrected )

Download Jenna's script (Add here) to clean up the file
To run the clean up script (???Should we explain what this script does???)
$ perl scriptname -i input file -o output file

Example: 

$ perl /Users/microBEnet/Desktop/RDPcleanup.pl -i /Users/microBEnet/Desktop/newTatumella.fa -o TatumellaRAST_RDP.fa

FastTree 
Go to
http://www.microbesonline.org/fasttree/#Install
and download the FastTree.c program by right clicking on it and saving the link

FastTree requires a gcc compiler to function? Which you can download and install from xcode here
https://developer.apple.com/downloads/index.action?q=xcode

In order to download xcode you will need to register as a developer with Apple which takes only a couple minutes.

After downloading xcode follow your computerâ€™s instructions to install it. Once it is installed open the program and open preferences (under the xcode tab). Click on the downloads option and install the command line tools. 

Once you have successfully downloaded and installed xcode, open the terminal, navigate to your downloads file (or wherever you are storing FastTree) and copy and paste the following syntax

$ gcc -O3 -finline-functions -funroll-loops -Wall -o FastTree FastTree.c -lm

To run FastTree, type the following in the terminal
$ FastTree -gtr -nt alignment.file > tree_file 

For Example
$ /Users/Madison/Downloads/FastTree -gtr -nt /Users/Madison/Desktop/newTatumella > /Users/Madison/Desktop/tree_file_tatumella.tre

The alignment file will be the output file of the cleanup script
And the tree_file output name should include a path to where you want it to be located. The output name should also end in .tre in order to be recognized by Dendroscope

Viewing the Tree
Download and install Dendroscope
http://ab.inf.uni-tuebingen.de/software/dendroscope/

You will need to obtain a license here
http://www-ab2.informatik.uni-tuebingen.de/software/dendroscope/register/

Enter the license into Dendrosope and then you can open your alignment to view it (file<open).
 
 

Once the tree is visible, the first step is to re-root the tree to the outgroup. Expand the tree by clicking the expansion button (labeled in the above screen shot), then scroll through the tree to locate the outgroup. Click on the beginning of the taxa name, to select it, and reroot the tree by going to edit and selecting re-root.  We recommend viewing the tree as a phylogram which can be accomplished by clicking on the phylogram button (labeled in the above screen shot). From this tree it should be possible to determine the phylogenetic placement of the candidate sequence, and in some cases to give it a name with more certainty than a simple BLAST search.  Below are examples of a relatively informative tree and a relatively uninformative tree:

TI
 
In tree shown above (genus Brachybacterium), our sample of interest from an assembly is "Brachybacterium muris UCD-AY4" (REF). It falls within a clade where every named member has the same name "Brachybacterium muris", and this name does not occur elsewhere on the tree. Hence, we were confident enough to name our sample as that species.
 
In the tree shown above (genus Microbacterium) our species of interest is Microbacterium sp. str. UCD-TDU (REF). In contrast to the Brachybacterium example, here our species falls within a poorly defined clade containing multiple species. In this case we did not assign a species name to this isolate.
