#Discussion
In an effort to democratize the process of microbial sequencing and de novo assembly, we have designed a workflow that would allow a small lab, one operating without a lab technician or bioinformatician, to take a sample and carry it through from swab to publication. There are many options for sequencing, assembling, and annotating microbial genomes.  This workflow is only one path through the many choices that could be made in a genome sequencing project. 

All of the scripts and programs for this workflow are open-source and available online for free to ensure that individual researchers and small groups are able to access and utilize the tools necessary to complete the workflow. In general, many available bioinformatic tools are free and open source, but the installation, operation and maintenance of these programs is often complex, requiring specific technical expertise or extensive detailed instructions and best practices in the field remain undefined.

It should also be noted that a sequenced genome is simply a starting place for analysis.  Analysis of the genome in detail, and comparative genomics are both outside the scope of this workflow.  These additional steps could include analyses such as aligning the genome to a close relative to look for rearrangements (e.g. Progressive Mauve REF), phylogenomic analysis (e.g. what?), looking for evidence of horizontal gene transfer (e.g. how?), looking for novel functions within the lineage (how?), examining genomic context (e.g. JContextExplorer, REF) and phylogenomic profiling (REF).  MORE STUFF HERE.


