#Organism Identification using 16s rRNA gene sequence
In a classroom or undergraduate research setting the project may not have a particular bacterial species in mind. In this case it is necessary to screen the 16S Sanger sequencing results for possible genome sequencing candidates.  We recommend starting with BLAST results, then continuing onto the Genomes Online Database (GOLD), and simply Google searching.  In many cases it will be handy to also build a phylogenetic tree to aid in identification since the 16S results may not be sufficiently informative.

##BLAST 16S rDNA sequence
Begin by navigating to the Standard Nucleotide BLAST at NCBI:

http://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn&PAGE\_TYPE=BlastSearch&LINK_LOC=blasthome

Paste in your consensus sequence.  We recommend checking the box to exclude Uncultured/environmental sample sequences since these will not be informative for identification. Be sure the nucleotide collection (nr/nt) is selected under database and click the BLAST button.

##Interpreting the results
Depending on the quality of the Sanger sequencing and the particular bacteria sequenced, the BLAST results can range from definitive to relatively uninformative. Examples of both are discussed below.

1. In Figure X, it is not necessary to build a phylogenetic tree for further identification. All of the top hits are either Kocuria rosea or Kocuria sp have e-values of 0.0, perfect query coverage, and 99% to100% identity. In this case, proceed to "Using GOLD".
2. In Figure Y the results are much more ambiguous. The results show more than 99% identity to multiple species within multiple genera. In this case, proceed to "Making a Phylogenetic Tree", before using GOLD.




