Feel free to add notes/critiques of the rocketship/summary figure here
The Figure above illustrates all of the steps of the workflow. A short desescription of each step and its equivalent ??? are listed below.
**Culture and Isolation**-Here Here we cover the steps necessary to take a sample through plating, dilution streaking, overnight growth, creating a glycerol stock, 16s PCR and preparation for Sanger sequencing.    
We assume a starting point of wanting to isolate an organism from a particular environment and needing to identify it.  Users starting with a known organism should proceed to "Library Preparation and Sequencing‚Äù.
Culture and Isolation covers steps 5 (Isolation) through 6.3 (PCR reaction)
**16S Sanger Sequencing**-We recommend sending the samples to a DNA sequencing facility.
**Alignment**-Upon receiving Sanger reads from a sequencing facility, it is necessary to quality trim the reads, reverse complement the sequences, align the reads and generate a consensus sequence. This is easiest to do visually via a chromatogram allowing the user to see the trace and process the sequences manually.
**I.D. Species**-In a classroom or undergraduate research setting the researchers may not have a particular bacterial species in mind.  In this case it is necessary to screen the 16S Sanger sequencing results for possible genome project candidates.   We recommend starting with the BLAST results, then continuing onto the Genomes Online Database (GOLD), and simply Google searching.   In many cases it will be handy to first build a phylogenetic tree to aid in identification since the 16S results may not be sufficiently informative.
**DNA Extraction**-In thks step, DNA is extracted from the species of interest.
